.github/workflows/self-hosted.yml

```
@@ -0,0 +1,161 @@
+        run: pnpm --filter @ecency/self-hosted test
+      - name: Run hosting API tests
+        run: cd apps/self-hosted/hosting/api && npm install && npm test
+
+  build-blog:
+    needs: tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
```

actionlint (1.7.11)

[error] the runner of "docker/login-action@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)